slackbot/dispatcher.py

```diff
@@ -22,6 +21,12 @@ def __init__(self, slackclient, plugins):
         self._pool = WorkerPool(self.dispatch_msg)
         self._plugins = plugins
 
+        alias_regex = ''
+        if hasattr(settings, 'ALIASES') and settings.ALIASES != '':
+            alias_regex = '|(?P<alias>{})'.format('|'.join([re.escape(s) for s in settings.ALIASES.split(',')]))
+
+        self.AT_MESSAGE_MATCHER = re.compile(r'^(?:\<@(?P<atuser>\w+)\>{}):? (?P<text>.*)$'.format(alias_regex))
+
```

Comment:
new stuff: needed to move this into the ```__init__``` method in order to allow unit tests to re-create AT_MESSAGE_MATCHER when we create and destroy MessageDispatcher() in py.text fixtures